src/config/tempo.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  token: process.env.TEMPO_TOKEN || '',
+  url: 'https://api.tempo.io/core/3/',
```

Comment:
Url can also be fetched from .env but by default it will take given url. It's just good practice.
```suggestion
  url: process.env.TEMPO_URL || 'https://api.tempo.io/core/3/',
```